scripts/tests/test_user_summary.py

```diff
@@ -51,10 +40,13 @@ def setUp(self):
         u.date_disabled = self.a_while_ago
         u.save()
 
-        modify_user_dates_in_mongo(self.yesterday)
+        for user in OSFUser.objects.all():
+            user.date_registered = self.yesterday
+            user.save()
+
 
     def tearDown(self):
-        User.remove()
+        OSFUser.objects.all().delete()
```

Comment:
Should no longer be necessary, since each test is run in a transaction which is rolled back at the end of the test.